src/views/UserProfileView/profileEditView/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ProfileEditComponent from '../../../components/Profile/ProfileEdit';
+import { profileUpdate, userImageUpload } from '../../../redux/actions/userProfileActions/actions';
+
+class profileEditView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bio: '',
+      image: '',
+    };
+  }
+
+
+  componentWillReceiveProps(nextProps) {
+    nextProps.history.push('/profiles');
+  }
+
+  onChange = (event) => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  onImageChange = (event) => {
+    this.setState({
+          [event.target.name]: event.target.files[0],
+        },
+        () => {
+          const imageData = {};
+          const { image } = this.state;
+          imageData.data = { image };
+          const user = JSON.parse(localStorage.getItem('user'));
+          const token = JSON.parse(localStorage.getItem('accessToken'));
+          const { username } = user;
+          imageData.token = token;
+          imageData.username = username;
+          const { userImageUpload: uploadImage } = this.props;
+          uploadImage(imageData);
+        });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { profileUpdate: profileUpdateAction } = this.props;
+    const user = JSON.parse(localStorage.getItem('user'));
+    // const token = JSON.parse(localStorage.getItem('accessToken'));
+    const { username } = user;
+    const updateData = {};
+    const { bio } = this.state;
+    updateData.data = { bio };
+    // updateData.token = token;
+    updateData.username = username;
+    profileUpdateAction(updateData);
+  };
+
+  render() {
+    // console.log(this.props);
```

Comment:
There is commented code on this file. Kindly remove them

Reply:
Hi, @johnwayodi 
Thank you for your attention to detail.
I have corrected by removing the line item.

Regards
Steven Ennis